mesheryctl/pkg/utils/helpers.go

```diff
@@ -950,6 +950,15 @@ func SetOverrideValues(ctx *config.Context, mesheryImageVersion string) map[stri
 		}
 	}
 
+	// disable the operator
+	if ok := ctx.GetDisableOperator(); ok {
+		if valueOverrides["env"] == nil {
```

Comment:
@abdullah1308 You misunderstood me
```suggestion
	if ctx.GetDisableOperator() {
		if _, ok := valueOverrides["env"]; ok {
```